proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -102,6 +102,9 @@ message GetMutableStateResponse {
     bool is_sticky_task_queue_enabled = 18;
     int64 last_first_event_txn_id = 19;
     string first_execution_run_id = 20;
+    // If using build-id based versioning: version stamp of last worker to process a workflow
+    // task for this workflow.
```

Comment:
Should these docstrings say "complete" rather than process? IE: Failing a workflow task shouldn't affect the latest version